src/wrappers/create-wrapper-array.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import WrapperArray from './wrapper-array'
+import type Wrapper from './wrapper'
+import type VueWrapper from './vue-wrapper'
+
+export default function createWrapperArray (wrappers: Array<Wrapper | VueWrapper>) {
```

Comment:
Can you write a few tests for this method and add validation (so check that Wrappers is an array of wrappers and throw an error if not)